x/stake/handler.go

```diff
@@ -71,6 +71,11 @@ func NewHandler(k keeper.Keeper, govKeeper gov.Keeper) sdk.Handler {
 			return handleMsgSideChainRedelegate(ctx, msg, k)
 		case types.MsgSideChainUndelegate:
 			return handleMsgSideChainUndelegate(ctx, msg, k)
+		case types.MsgSideChainStakeMigration:
+			if !sdk.IsUpgrade(sdk.FirstSunsetFork) {
```

Comment:
`MsgSideChainStakeMigration` tx should be disable after the `Second Sunset Fork`

Reply:
Done